scene/resources/surface_tool.cpp

```diff
@@ -252,13 +260,8 @@ void SurfaceTool::set_weights(const Vector<float> &p_weights) {
 	last_weights = p_weights;
 }
 
-void SurfaceTool::add_smooth_group(bool p_smooth) {
-	ERR_FAIL_COND(!begun);
-	if (index_array.size()) {
-		smooth_groups[index_array.size()] = p_smooth;
-	} else {
-		smooth_groups[vertex_array.size()] = p_smooth;
-	}
+void SurfaceTool::set_smooth_group(uint32_t p_group) {
+	last_smooth_group = p_group;
```

Comment:
Currently this variable seems to be set but never queried. Since it's a private member, it currently looks like dead code.

`void SurfaceTool::generate_normals(bool p_flip)` doesn't seem to do normals smoothing anymore.

Is this intended ?

Reply:
no, I guess I messed up, thanks for taking a look!

Reply:
ok fixed